index.js

```diff
@@ -37,6 +32,12 @@ module.exports = function (path, second, third) {
   // ... else proceed as usual
   asciify_core(path, opts, callback || console.log);
 }
+// Set of basic characters ordered by increasing "darkness"
+// Used as pixels in the ASCII image
+asciify.chars = ' .,:;i1tfLCG08@';
+asciify.num_c = asciify.chars.length - 1;
```

Comment:
I'm okay with making this configurable but I'd prefer that they be explicit options. Perhaps add them as the `opts` arg to the main `module.exports`?

Alternately, if you'd like to make this a permanent setting (i.e. set the chars once, then use the lib several times with that set) let's create setters and getters for local vars